src/TitleDetails/TitleDetails.js

```diff
@@ -293,34 +295,48 @@ const TitleDetails = ({
   } = useFilters(noop);
   const { filters: boundItemsFilters } = useFilters(noop, { [MENU_FILTERS.bound]: ['true'] });
 
-  const expectedPiecesActions = useMemo(
-    () => (
-      <TitleDetailsExpectedActions
-        applyFilters={applyExpectedPiecesFilters}
-        filters={expectedPiecesFilters}
-        hasReceive={hasReceive}
-        onPieceCreate={onPieceCreate}
-        openReceiveList={onReceivePieces}
-        titleId={titleId}
-        disabled={isPiecesLock || restrictions?.protectUpdate}
-        canAddPiece={!restrictions?.protectCreate}
-        toggleColumn={toggleExpectedPiecesColumn}
-        visibleColumns={expectedPiecesVisibleColumns}
-      />
+  const expectedPiecesProtectedActions = useMemo(() => ({
+    [EXPECTED_PIECES_ACTION_NAMES.addPiece]: (
+      restrictions?.protectCreate
+      || (isOrderPending && poLine.checkinItems === false)
```

Comment:
Just for information, BE has the same validation rule to protect API users as well:
https://github.com/folio-org/mod-orders/pull/1068/files#diff-985428c1f8fca514c862e5313f72e1e5e22f9aff6f1d16f8bdede16e210f829bR49